### ISSUE TITLE
BRAIN TRUST IMAGES

### ISSUE THREAD

**Issue by vxsm26** [GCGT/TemboGroup#24]
Subject
Brain trust page images

To complete
We need the two images for everybody in this page the one with the name and the one of the actual person

**Comment by vxsm26** [comment]
Hey @ZhangJieSong I think we can close this one now with the last update you made right?

**Comment by ZhangJieSong** [comment]
Yeah, you should have 3 different versions of images for each Braintrust member. Remember that one version is just in case you need it - the darkened image without text. It's like a backup, in case, for some wild reason, you might have need for these.
